Pellicola/Classes/PellicolaPresenter.swift

```diff
@@ -167,8 +175,14 @@ public final class PellicolaPresenter: NSObject {
                                         imagesDataFetcher: imagesDataFetcher,
                                         albumData: album)
         let assetsViewController = AssetsViewController(viewModel: viewModel, style: style)
-        assetsViewController.didSelectImages = { [weak self] images in
-            self?.dismissWithImages(images)
+        
+        assetsViewController.didStartProcessingImages = { [weak self] in
+            self?.didStartProcessingImages?()
+        }
+        
+        assetsViewController.didFinishProcessingImages = { [weak self] urls in
+            self?.didFinishProcessingImages?(urls)
```

Comment:
would it make sense to call didFinishProcessingImages in the navigationcontroller dismiss completion?

Reply:
Ok